docs/development_suite/overview-dev-suite.md

```diff
@@ -35,3 +54,5 @@ Mia-Platform Console is organized into hierarchical levels to help users navigat
 - **Project Sections:** Designated areas within a project that delineate and organize functionalities and tasks.
```

Comment:
```suggestion
- **Companies:** An organizational entity that acts as a container for projects. It can represent the entire organization or a specific operational unit.
- **Projects:**A structured collection of microservices, configurations, and data within a Company. It might represent a digital system, a specific component or a Package Business Capability (PBC).
- **Project Sections:** Designated areas within a project that define and organize functionality and tasks.

```

Reply:
It should be nice have an Image of this structure

Reply:
Hi @Massibian, your feedbacks are ok but at the moment there is no image for the structure.